lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOMembershipInvtnSubmissionDAOImpl.java

```diff
@@ -292,5 +292,20 @@ public void deleteByTeamAndUser(long teamId, long inviteeId)
 		namedJdbcTemplate.update(DELETE_INVITATIONS_BY_TEAM_AND_INVITEE, param);
 	}
 
+	@Override
+	public String getInviteeEmail(String membershipInvitationId) {
+		MapSqlParameterSource param = new MapSqlParameterSource();
+		param.addValue(COL_MEMBERSHIP_INVITATION_SUBMISSION_ID, membershipInvitationId);
+		return namedJdbcTemplate.queryForObject(SELECT_INVITEE_EMAIL, param, String.class);
+	}
 
+	@WriteTransactionReadCommitted
+	@Override
+	public int updateInviteeId(String misId, long inviteeId) {
+		MapSqlParameterSource param = new MapSqlParameterSource();
+		param.addValue(COL_MEMBERSHIP_INVITATION_SUBMISSION_INVITEE_ID, inviteeId);
+		param.addValue(COL_MEMBERSHIP_INVITATION_SUBMISSION_ID, misId);
+		param.addValue(COL_MEMBERSHIP_INVITATION_SUBMISSION_ETAG, UUID.randomUUID().toString());
+		return namedJdbcTemplate.update(UPDATE_INVITEE_ID, param);
```

Comment:
return type -> void
if number updated == 0 throw NotFoundException (misId is not found) or IllegalArgumentException.
if number updated > 1 then throw DatastoreException, which maps to a 500 status.